plugins/orchestrator-backend/src/service/router.ts

```diff
@@ -324,6 +333,44 @@ function setupInternalRoutes(
     },
   );
 
+  // v1
+  router.get('/workflows/:workflowId/source', async (req, res) => {
```

Comment:
seemse like some issue with rebase, this reverts a change from a previous PR

Reply:
fixed